runners/samza/src/main/java/org/apache/beam/runners/samza/SamzaPipelineOptions.java

```diff
@@ -173,4 +173,11 @@ public ExecutorService create(PipelineOptions options) {
           new ThreadFactoryBuilder().setNameFormat("Process Element Thread-%d").build());
     }
   }
+
+  @Description(
```

Comment:
Do we still need this option?

Reply:
yes. This option is used in logics from beam-runner: https://linkedin.ghe.com/multiproduct/beam-runner/pull/113.

I wanted to keep it which gives us a way to disable the change in the beam-runner for unexpected issues.